### ISSUE TITLE
Ansible playbook fails installing dependencies in Fedora Linux 36 (Workstation Edition)

### ISSUE THREAD

**Issue by jphein** [althea-net/althea-firmware#65]
```
[jp@jp-Latitude-7280 althea-firmware]$ ansible-playbook first-time-setup.yml -bK
BECOME password: 
[WARNING]: provided hosts list is empty, only localhost is available. Note that
the implicit localhost does not match 'all'

PLAY [127.0.0.1] ***************************************************************

TASK [Gathering Facts] *********************************************************
ok: [127.0.0.1]

TASK [check-deps : Install Debian deps] ****************************************
skipping: [127.0.0.1]

TASK [check-deps : Install Ubuntu deps] ****************************************
skipping: [127.0.0.1]

TASK [check-deps : Install Fedora deps] ****************************************
fatal: [127.0.0.1]: FAILED! => {"changed": false, "failures": ["No package libselinux-python available."], "msg": "Failed to install some of the specified packages", "rc": 1, "results": []}

PLAY RECAP *********************************************************************
127.0.0.1                  : ok=1    changed=0    unreachable=0    failed=1    skipped=2    rescued=0    ignored=0   


```

**Comment by jkilpatr** [comment]
Fixed this in 02b4eb3ebd9644f42857bcaa6e92533943d65fb6

thanks for letting me know, that code is a little stale I've opened a ticket to have it run in ci.

**Comment by jphein** [comment]
Thank you so much! =]
